go/worker/compute/txnscheduler/committee/node.go

```diff
@@ -2,12 +2,12 @@ package committee
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"sync"
 	"time"
 
 	"github.com/opentracing/opentracing-go"
+	"github.com/pkg/errors"
```

Comment:
Don't introduce a new dependency on a package we are trying to remove. Put the other errors back.

Reply:
Ooops, some IDE magic again :thinking: